lib/Service/ReferenceFacesFinderService.php

```diff
@@ -69,6 +74,9 @@ public function __construct(LoggerInterface $logger, IManager $contacts, IUserSe
      * @return array
      */
     public function findReferenceFacesForUser(string $userId):array {
+        if ($this->config->getAppValue('recognize', 'faces.enabled', 'false') !== 'true') {
+            return 0;
```

Comment:
This violates the type hints. We should rather use `[]` here.

Reply:
Generally, I feel like the check does not belong in this class.

Reply:
@marcelklehr I didn't like this much either, but not sure where else this should be done as this is called from the Service and the Command and I didn't want to duplicate some code